git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
I'm a +1 for structured commit messages. In fact, for the Progressive Web SDK, we are moving to a very strict format for commit messages:

```
<type>(<scope>): <subject>
<BLANK LINE>
<body>
<BLANK LINE>
<footer>
```

Based on that, we will auto-generate the `CHANGELOG.md` file as well as automatically create a release and publish it to NPM and Github. This is needed in order to move closer to continuous integration.

Regardless of this upcoming change in our release process, I believe we're better off not relying on the Jira <> Github integration. Most often when it comes to maintenance, I open Github and check a files history to figure out what happened.